pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/filter/AndOrFilterSeparator.java

```diff
@@ -4,31 +4,33 @@
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 import org.broadinstitute.dsm.model.Filter;
 
 //class to separate 'AND' or 'OR' conditions into type of Map<String, List<String>>
 public class AndOrFilterSeparator {
 
+    private static final String AND_OPERATOR = "AND";
```

Comment:
I added these to replace `AND_TRIMMED`, since those terms were distracting. The fact that the strings were trimmed is irrelevant here, and there is no efficiency in reusing constant terms.